components/typography/Paragraph.jsx

```diff
@@ -0,0 +1,8 @@
+export default const ParagraphName = ({children}) => { // ComponentName - будущее название компонента 
```

Comment:
```suggestion
export default const Paragraph = ({children}) => {
```
Не подчистил лишние комментарии и неверно именовал компонент